### ISSUE TITLE
Using CDN minified version of RouteNamePlugin doesnt work.

### ISSUE THREAD

**Issue by mocodev** [swup/route-name-plugin#8]
I'm not using a bundler in my project and am currenlty trying to upgrade Swup form v2 to v3. I need to use the RouteNamePlugin but including the CDN link `<script src="https://unpkg.com/@swup/route-name-plugin@2"></script>` results in js errors.

```
Uncaught ReferenceError: exports is not defined`
Uncaught TypeError: SwupRouteNamePlugin is not a constructor
```

Looking at the source of the CDN link, it doesnt appear to be a minified version and is still reliant on imports.

**Comment by mocodev** [comment]
After looking further it looks like the reference to the CDN link might be wrong in the documentation. `<script src="https://unpkg.com/@swup/route-name-plugin@3"></script>` brings up a minified version, but still getting js errors.

```
const swup = new Swup({
	linkSelector: 'a.btn',
	animateHistoryBrowsing: true,
    plugins: [
		new SwupBodyClassPlugin(),
		new SwupDebugPlugin(),
	    new SwupPreloadPlugin(),
	    new SwupRouteNamePlugin({ paths: true })
  	  ]
});
```

Results in

`Uncaught TypeError: class constructors must be invoked with 'new'` in reference to `https://unpkg.com/@swup/route-name-plugin@3`

**Comment by daun** [comment]
Is the swup CDN version working for you? They're technically using the same bundler.

**Comment by mocodev** [comment]
I'm currenlty using the Swup v3 CDN link `<script src="https://unpkg.com/swup@3"></script>` and is loading and initializing just fine. It's only when I introduce the RouteNamePlugin where things break.

**Comment by daun** [comment]
I've pushed an attempted fix as 3.0.1. Let me know if that fixes it.

**Comment by mocodev** [comment]
3.0.1 from CDN working as expected!
